<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3">
    <item>
     <layout class="QVBoxLayout" name="engrus_verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="engrusFileLineEdit">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string>engrus_input.txt</string>
          </property>
          <property name="placeholderText">
           <string>Название файла анг-рус слова</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="engrusFilePushButton">
          <property name="text">
           <string>Считать</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="engrusTableWidget">
        <property name="columnCount">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="engbelrus_verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="engbelrusFileLineEdit">
          <property name="text">
           <string>engbelrus_input.txt</string>
          </property>
          <property name="placeholderText">
           <string>Название файла анг-бел-рус слова</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="engbelrusFilePushButton">
          <property name="text">
           <string>Считать</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="engbelrusTableWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
